pkg/cloud-provider/cloudapi/azure/azure_crd_helpers.go

```diff
@@ -157,6 +151,47 @@ func computeInstanceToInternalVirtualMachineObject(instance *virtualMachineTable
 	return vmObj
 }
 
+// importTags function returns the tags which comply with kubernetes labels format.
+func importTags(vmTags map[string]*string) map[string]string {
```

Comment:
create a directory in util call k8s and move this import tags there.. wait for code refactor to get checked-in first. @Anandkumar26 I believe this is right helper for new k8s dir?

Reply:
Yes. This should be put in pkg/util/k8s package